SingularityService/src/main/java/com/hubspot/singularity/smtp/SingularityMailer.java

```diff
@@ -477,6 +497,10 @@ public void sendRequestScaledMail(SingularityRequest request, Optional<Singulari
   }
 
   public void sendRequestRemovedMail(SingularityRequest request, Optional<String> user, Optional<String> message) {
+    if (!maybeSmtpConfiguration.isPresent()) {
```

Comment:
Ah, fair enough. yeah, can put it at the beginning of the public methods for readability